src/main/resources/application.properties

```diff
@@ -1 +1,11 @@
 management.endpoints.web.exposure.include=info, health
+
+# datasource
+spring.datasource.url=jdbc:mysql://localhost:3307/{{DATABASE's NAME HERE}}?createDatabaseIfNotExist=true
```

Comment:
@mcruzdev what is `{{DATABASE's NAME HERE}}` ? I have the feeling that will fail if the user doesn't change that to something.